### ISSUE TITLE
Change border radius

### ISSUE THREAD

**Issue by itaysharir** [FelixKratz/yabai#18]
Is it possible to change the border radius on fyabai? ```yabai -m config border_radius 0``` doesn't work.

**Comment by itaysharir** [comment]
and ```yabai -m config windows_border_radius 0``` returns ```changing border radius is deactived.```

**Comment by FelixKratz** [comment]
I have killed changing border_radius here:
https://github.com/FelixKratz/yabai/commit/7016e286922905ce35135f4f5390b23585ba6270
simply drop the contents of this patch for src/message.c then it will work again. I did it because this is the default corner radius of macOS.

**Comment by itaysharir** [comment]
Sorry for the bother but just change back the lines that changed in the link you sent?

**Comment by FelixKratz** [comment]
Just uncomment this line:
https://github.com/FelixKratz/yabai/blob/7016e286922905ce35135f4f5390b23585ba6270/src/message.c#L1230
and delete the line below it

**Comment by itaysharir** [comment]
And uninstalled fyabai from brew

**Comment by itaysharir** [comment]
I uninstalled fyabai from brew, then ran ```sudo make clean install``` but when i then run ```./bin/yabai --start-service``` i get this: 
```
❯ ./bin/yabai --start-service
Load failed: 5: Input/output error
Try running `launchctl bootstrap` as root for richer errors.```

**Comment by itaysharir** [comment]
❯ sudo launchctl bootstrap
Usage: launchctl bootstrap <domain-target> [service-path, service-path2, ...]
    --angel
        Bootstrap the plist as a LaunchAngel

**Comment by FelixKratz** [comment]
I think you need to:
```bash
make install
cp bin/yabai /opt/homebrew/bin/yabai
yabai --stop-service
yabai --start-service
```

**Comment by itaysharir** [comment]
I ran:
```
make install
cp bin/yabai /opt/homebrew/bin/yabai
```
Output:
```
rm -rf ./bin
xcrun clang ./src/osax/payload.m -shared -fPIC -O2 -mmacosx-version-min=11.0 -arch x86_64 -arch arm64e -o ./src/osax/payload -F/System/Library/PrivateFrameworks -framework SkyLight -framework Foundation -framework Carbon
xcrun clang ./src/osax/loader.m -O2 -mmacosx-version-min=11.0 -arch x86_64 -arch arm64e -o ./src/osax/loader -framework Cocoa
xxd -i -a ./src/osax/payload ./src/osax/payload_bin.c
xxd -i -a ./src/osax/loader ./src/osax/loader_bin.c
rm -f ./src/osax/payload
rm -f ./src/osax/loader
mkdir -p ./bin
xcrun clang src/manifest.m src/osax/payload_bin.c src/osax/loader_bin.c -std=c99 -Wall -DNDEBUG -O2 -fvisibility=hidden -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64 -arch arm64 -F/System/Library/PrivateFrameworks -framework Carbon -framework Cocoa -framework CoreServices -framework SkyLight -o bin/yabai
```
I ran:
```
yabai --stop-service
yabai --start-service
```
Output:
```
Load failed: 5: Input/output error
Try running `launchctl bootstrap` as root for richer errors.
```

**Comment by itaysharir** [comment]
(All i did was uncoment line 1230 and comment line 1231)

**Comment by FelixKratz** [comment]
try:
```bash
make install
codesign --force -s - /bin/yabai
cp bin/yabai /opt/homebrew/bin/yabai
yabai --stop-service
yabai --start-service
```
maybe code signature is required.

**Comment by itaysharir** [comment]
```
rm -rf $HOME/Library/LaunchAgents/com.koekeishiya.yabai.plist
make install
codesign --force -s - bin/yabai
cp bin/yabai /opt/homebrew/bin/yabai
yabai --stop-service
yabai --start-service
```
Returned:
```
rm -rf ./bin
mkdir -p ./bin
xcrun clang src/manifest.m src/osax/payload_bin.c src/osax/loader_bin.c -std=c99 -Wall -DNDEBUG -O2 -fvisibility=hidden -mmacosx-version-min=11.0 -fno-objc-arc -arch x86_64 -arch arm64 -F/System/Library/PrivateFrameworks -framework Carbon -framework Cocoa -framework CoreServices -framework SkyLight -o bin/yabai
bin/yabai: replacing existing signature
yabai: service file '/Users/itaysharir/Library/LaunchAgents/com.koekeishiya.yabai.plist' is not installed! abort..
yabai: service file '/Users/itaysharir/Library/LaunchAgents/com.koekeishiya.yabai.plist' is not installed! attempting installation..
Load failed: 5: Input/output error
Try running `launchctl bootstrap` as root for richer errors.
```

**Comment by itaysharir** [comment]
Wait i have no idea what happened but suddenly my windows started tiling so it's working i guess

**Comment by itaysharir** [comment]
Is is possible to change the radius in the code (permanently) and then animations would work?

**Comment by FelixKratz** [comment]
Can you rephrase what you mean by "then animations would work"? Animations should work anyways.
